db/migrate/20231006063440_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :AuthorId
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] For the posts table here as well please make sure to include all the attributes specified in the ER diagram like the `title`, `text`, `comments_counter`, & `likes_counter`.
- [ ] As specified in the project requirements in `posts` table the column holding the foreign key to the `users` table should be named `author_id` so I'd suggest updating the code at line `4` to match that which should instead be `t.references :author, foreign_key: { to_table: 'users' }` this way you can remove the code at line `5`.